unlock-app/src/utils/durations.js

```diff
@@ -82,10 +82,33 @@ export function expirationAsDate(timestamp) {
     return durationsAsTextFromSeconds(timestamp)
 
   let expirationDate = new Date(0)
-  expirationDate.setUTCSeconds(timestamp)
+  expirationDate.setUTCSeconds(timestamp / 1000)
```

Comment:
See the error in `duration.test.js` - it looks like something is dividing by 1000 one too many times (and resulting in January 20, 1970, which is roughly July 7, 2022 divided by 1000).